.eslintrc.cjs

```diff
@@ -159,7 +161,9 @@ const REACT_RULES = {
 };
 
 const TYPESCRIPT_RULES = {
-  '@typescript-eslint/ban-types': WARN,
+  '@typescript-eslint/no-empty-object-type': WARN,
```

Comment:
These changes are needed due to dependency updates

Reply:
there are way around these checks. How are these for sure needed?

Reply:
I just updated these rules because I updated the version of eslint, and the rules I removed were no longer supported in the new version of eslint, so I replaced them with the new versions.